chart/flux/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: "1.17.0"
-version: 1.0.0
+version: 1.1.0
```

Comment:
Please undo this change, the version bump happens at release time.

Reply:
Right. Fixed now